frontend/src/components/panels/actionitems/ActionItemReason.tsx

```diff
@@ -0,0 +1,42 @@
+import { Dialog, IconButton, Stack, Typography } from '@mui/material';
+import { Close as CloseIcon } from '@mui/icons-material';
+import { IActionItem } from 'shared-types';
+
+interface ActionItemReasonProps {
+  open: boolean;
+  item: IActionItem | undefined;
+  onClose: () => void;
+}
+
+const ActionItemReason: React.FC<ActionItemReasonProps> = ({ open, item, onClose }) => (
+  <Dialog PaperProps={{ sx: { borderRadius: '1rem' } }} fullWidth onClose={onClose} open={open}>
+    <Stack minHeight="30vh" display="flex" direction="column" paddingLeft={5} paddingRight={4} paddingTop={8}>
```

Comment:
Bug: the content of the dialog had no padding on the right and clashed with the borders
Solution: added paddingRight